x-pack/platform/plugins/shared/onechat/server/services/tools/builtin/definitions/create_visualization/utils.ts

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { JsonSchema } from './types';
+
+export const fixArraySchemas = (schema: JsonSchema): JsonSchema => {
```

Comment:
I assume this is used to fix map's messy schemas? Are they aware of that / planning to fix it upstream?

Reply:
i think i can remove it, its left over from my testing of withStructuredOutput and conversion between joi-json-zod